templates/vsc/ts/notification-express/src/notification/storage.ts

```diff
@@ -0,0 +1,137 @@
+// Copyright (c) Microsoft Corporation.Add commentMore actions
+// Licensed under the MIT license.
+
+import { ConversationReference } from "@microsoft/agents-activity";
+import * as fs from "fs";
+import * as path from "path";
+
+/**
+ * Represents a page of data.
+ */
+export interface PagedData<T> {
+  /**
+   * Page of data.
+   */
+  data: T[];
+
+  /**
+   * The Continuation Token to pass to get the next page of results.
+   *
+   * @remarks
+   * Undefined or empty token means the page reaches the end.
+   */
+  continuationToken?: string;
+}
+
+export class LocalConversationReferenceStore {
```

Comment:
minor: I remember BotFramework has some kind of `IStorage` interface, which we can use to implement the local store. Did agent SDK provides similar interface for us to use?

Reply:
Add an interface to extend Agents sdk Storage interface (as it doesn't have list method).